internal/controller/pipelinerollout_controller.go

```diff
@@ -519,13 +525,7 @@ func (r *PipelineRolloutReconciler) shouldBePaused(ctx context.Context, pipeline
 
 	// If there is a need to update, does it require a pause?
 	// Here we look for whether we are trying to update Pipeline, or whether we are already processing an update which required us to pause
-	var pipelineUpdateRequiresPause bool
-	if pipelineNeedsToUpdate || pipelineUpdating {
-		pipelineUpdateRequiresPause, err = needsPausing(existingPipelineDef, newPipelineDef)
-		if err != nil {
-			return nil, err
-		}
-	}
+	pipelineUpdateRequiresPause := pipelineNeedsToUpdate || pipelineUpdating
```

Comment:
nit: the name `pipelineUpdateRequiresPause` is a bit misleading now (since I gather all updates in this function require pausing) - perhaps we no longer need this variable set and can just include`pipelineNeedsToUpdate || pipelineUpdating` directly in the setting of `shouldBePaused` below?

Reply:
Good call, will make that change.